web/src/components/troubleshoot/TroubleshootContainer.tsx

```diff
@@ -9,13 +9,12 @@ import EditRedactor from "../redactors/EditRedactor";
 
 // Types
 import { App } from "@types";
-import { RouteComponentProps } from "react-router-dom";
 
 type Props = {
-  app: App;
+  app: App | null;
   appName: string;
 };
-class TroubleshootContainer extends Component<Props & RouteComponentProps> {
```

Comment:
no longer need to have the react router props type definition on this component.